pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -94,11 +94,9 @@ self: super: builtins.intersectAttrs super {
   # Won't find it's header files without help.
   sfml-audio = appendConfigureFlag super.sfml-audio "--extra-include-dirs=${pkgs.openal}/include/AL";
 
-  cachix = overrideCabal (addBuildTools (enableSeparateBinOutput super.cachix) [pkgs.boost]) (drv: {
-    postPatch = (drv.postPatch or "") + ''
-      substituteInPlace cachix.cabal --replace "c++14" "c++17"
-    '';
-  });
+  cachix = addBuildTools (enableSeparateBinOutput super.cachix) [pkgs.boost];
```

Comment:
What part of `boost` does `cachix` need? Could you please document that in a comment?

Reply:
See 821676200d6f65c9eb4584e393af1204eeb7e363